src/main/java/wolox/training/utils/VariablesConstants.java

```diff
@@ -8,4 +8,7 @@
 
   //VARIABLES ON USERS TEST
   public static String BOOK_URL = "/api/books/";
+
+  //EXTERNAL SERVICE
+  public static String OPEN_LIBRARY_URL = "https://openlibrary.org/api/books?bibkeys=ISBN:%s&format=json&jscmd=data";
```

Comment:
`https://openlibrary.org/api` should be a property. Use `@Value` https://www.baeldung.com/spring-value-annotation

Reply:
Leave `/books?bibkeys=ISBN:%s&format=json&jscmd=data` as a constant